app/javascript/components/molecules/ExpandableTable.js

```diff
@@ -71,12 +74,13 @@ class ExpandableTable extends React.Component {
   }
 }
 
-ExpandableTable.ExpanderButton = ({ onClick, label = 'Expand' }) => {
+ExpandableTable.ExpanderButton = ({ onClick, label = 'Expand', id }) => {
   return (
     <button
       type='button'
       className='button button-link action-link'
-      onClick={onClick}>
+      onClick={onClick}
+      id={id !== null ? id : null}>
```

Comment:
@akegan are you anticipating a case here where `id` would have a truthy value that we wouldn't consider suitable?

Reply:
as in, if `id` is `false` or `undefined`, do we not consider that suitably falsey here for some reason?

Reply:
nope, I updated some of the other ones but not this one. I'll update it now